typescript/syntax/Statements.sdf3

```diff
@@ -2,19 +2,28 @@ module Statements
 
 imports
 
+  Common
   Declarations
   Expressions
+  Functions
+ 
+context-free priorities
+
+  ReturnStatement.value > ReturnStatement.empty
 
 context-free syntax
 
   Statement.variableStatement = VariableStatement
   Statement.return = ReturnStatement
-  Statement.expression = <<PrimaryExpression>;>
+  Statement.expression = <<ExpressionInStatement><SemiColon?>>
+  
+  ExpressionInStatement = PrimaryExpression
+  ExpressionInStatement = FunctionDeclaration {reject}
```

Comment:
@eelcovisser By putting this in a different construct and then reject one specific `PrimaryExpression`, it works :smile: